templates/site/datacatalog.html

```diff
@@ -23,8 +23,33 @@
     {% with %}
         <script>
             function nodeFilter([_name, data]) {
-                return data.services && data.services.some(service => service.keywords.includes("catalog"))
+                return data.services && data.services.some(service => service.name.includes("stac-browser"))
             }
+
+            function setNodeLink(name, data, node_id_suffix) {
+
+                data.services.forEach((service) => {
+                    console.log(service)
+                    console.log(service.name)
+
+                    if(service.name === "stac-browser")
+                    {
+                        if(service.keywords.includes("catalog")) {
```

Comment:
Why are we checking this too?

Reply:
It was from the previous iteration.

Reply:
Removed the condition checking for "catalog" in keywords